auditor/src/client/mod.rs

```diff
@@ -172,17 +239,28 @@ impl AuditorClient {
         Ok(())
     }
 
+    /// Gets all records from the Auditors database.
+    ///
+    /// # Errors
+    ///
+    /// * [`ClientError::ReqwestError`] - If there was an error sending the HTTP request.
     #[tracing::instrument(name = "Getting all records from AUDITOR server.", skip(self))]
-    pub async fn get(&self) -> Result<Vec<Record>, reqwest::Error> {
+    pub async fn get(&self) -> Result<Vec<Record>, ClientError> {
         self.client
             .get(&format!("{}/get", &self.address))
             .send()
             .await?
             .error_for_status()?
             .json()
             .await
+            .map_err(ClientError::ReqwestError)
```

Comment:
Can this be removed? I imagine that removing this will cause a type error because of the lack of `?`. Alternatively, you can use `?` and wrap the entire expression in `Ok(...)`.

Reply:
`Ok(...?)` worked. Fixed.